code/modules/surgery/organic.dm

```diff
@@ -446,7 +448,7 @@
 		SPAN_NOTICE("You close the wounds on [organ.get_surgery_name()] with \the [tool].")
 	)
 	organ.stopBleeding()
-	organ.take_damage(0, max(tool.force/5, tool.force/(user.stats.getStat(STAT_BIO)/STAT_LEVEL_BASIC))) //So it's a bad idea to remove shrapnel with a chainsaw
+	organ.take_damage(0, tool.force/0.3)
```

Comment:
```suggestion
	organ.take_damage(tool.force / 0.3, 0)
```
No idea why it was inversed in the first place, this was doing 0 damage since the first argument is the amount of damage and the second one is if yes or no the `take_damage` is silent.

Reply:
The first one is brute damage, the second one is burn damage, according to the proc

Reply:
Ah you're right, I was looking at `organ/internal/take_damage`, my bad.